flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/TemporalTypesTest.scala

```diff
@@ -37,6 +37,17 @@ import java.util.{Locale, TimeZone}
 
 class TemporalTypesTest extends ExpressionTestBase {
```

Comment:
Could you extract all `Timestamp` related test to a dedicated one? `TemporalTypesTest` seems weird.

Reply:
So divide it to TimestampTypeTest/DateTypeTest/TimeTypeTest ?

Reply:
That would be great.

Reply:
Double checked the `TemporalTypesTest` and find some datetime function can handle multiple datetime types(e.g. extract/floor etc). Split the class causes tests for one single function live in different places. So I would rename this as DateTimeTypesTest to avoid confusing.